packages/eslint-plugin-lit-a11y/lib/rules/valid-lang.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @fileoverview Ensures the document has a valid `lang` attribute.
+ * @author open-wc
+ */
+
+const ruleExtender = require('eslint-rule-extender');
+const tags = require('language-tags');
+const { TemplateAnalyzer } = require('../../template-analyzer/template-analyzer');
+const { elementHasAttribute } = require('../utils/elementHasAttribute');
+const { HasLitHtmlImportRuleExtension } = require('../utils/HasLitHtmlImportRuleExtension');
+const { isHtmlTaggedTemplate } = require('../utils/isLitHtmlTemplate');
+
+const ValidLangRule = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Ensures the document has a valid `lang` attribute.',
+      category: 'Accessibility',
+      recommended: false,
+      url:
+        'https://github.com/open-wc/open-wc/blob/master/packages/eslint-plugin-lit-a11y/docs/rules/valid-html.md',
+    },
+    messages: {
+      noLangPresent: 'No lang attribute is present.',
+      invalidLang: 'The value passed through to lang is not BCP 47 compliant.',
+    },
+    fixable: null,
+    schema: [],
+  },
+
+  create(context) {
+    return {
+      TaggedTemplateExpression(node) {
+        if (isHtmlTaggedTemplate(node, context)) {
+          const analyzer = TemplateAnalyzer.create(node);
+
+          analyzer.traverse({
+            enterElement(element) {
+              const loc = analyzer.getLocationFor(element);
+
+              if (element.name === 'html') {
+                if (!elementHasAttribute(element, 'lang')) {
+                  return context.report({
+                    loc,
+                    messageId: 'noLangPresent',
+                  });
+                }
+
+                const { lang } = element.attribs;
+
+                if (!tags.check(lang)) {
+                  return context.report({
+                    loc,
```

Comment:
If an invalid language is passed to the `lang` attribute, it'd be nice to report the error on the attribute only, instead of on the html tag. You can use `analyzer.getLocationForAttribute` to get the location of the `lang` attribute